backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/Sanitize.kt

```diff
@@ -23,6 +23,10 @@ class Sanitize(private val logger: Logger, httpClient: HttpClient) : PipelineSte
         val stateBbox = featureFetcher.queryFeatures(listOf(Pair("state", STATE))).first().bbox
 
         input.map { it.sanitize() }.filter {
+            if (it.longitude == null || it.latitude == null) {
+                logger.info("'${it.name}, ${it.location}' was filtered out because longitude or latitude are null")
+                return@filter false
+            }
```

Comment:
So does this mean, we filter them out at a later point if they make it to the "Sanitize" step?

Reply:
We filter locations without coordinates here instead of in the `Filter` step

Reply:
Then the title of the PR is wrong? We simply delay filtering accepting stores without coordinates, right?

Reply:
Only kind of. Stores without coordinates are first sanitized and only if they still have no coordinates afterwards filtered.